dev/table.h

```diff
@@ -132,7 +132,12 @@ namespace sqlite_orm {
                 });
                 return res;
             }
-            
+
+			template<typename ImplType = table_impl<Cs...>>
+			typename ImplType::composite_key_type get_composite_key() {
```

Comment:
where this function is used? If nowhere please remove it

Reply:
@fnc12  resolved in #329